modules/gpu/src/pyrlk.cpp

```diff
@@ -98,7 +107,9 @@ namespace
     }
 }
 
-void cv::gpu::PyrLKOpticalFlow::sparse(const GpuMat& prevImg, const GpuMat& nextImg, const GpuMat& prevPts, GpuMat& nextPts, GpuMat& status, GpuMat* err)
+void cv::gpu::PyrLKOpticalFlow::sparse(const GpuMat& prevImg,
+						const GpuMat& nextImg, const GpuMat& prevPts,
+						GpuMat& nextPts, GpuMat& status, GpuMat* err)
```

Comment:
Don't use "tabs"

Reply:
Ok, I'm going to change pthreads by c++11 std::mutex and std::condition_variable and add accuracy tests and performance tests

Reply:
C++11 is not allowed for OpenCV 2.4/3.x - C++98 is required.

Reply:
I understand, so I can use TBB (tbb::mutex and condition_variable), and make compilable the thread-safe version only if there is TBB support
"#ifdef HAVE_TBB"
What do you think?
Thank you for your help